app/models/family.rb

```diff
@@ -41,7 +41,7 @@ class Family < ApplicationRecord
     id guardian_first_name guardian_last_name guardian_zip_code guardian_country
     guardian_phone agency_guardian_id home_adult_count home_child_count home_young_child_count
     sources_of_income guardian_employed guardian_employment_type guardian_monthly_pay
-    guardian_health_insuranceb comments created_at updated_at partner_id military
+    guardian_health_insurance comments created_at updated_at partner_id military
```

Comment:
```suggestion
    guardian_health_insuranceb comments created_at updated_at partner_id military
```

Just to be absolutely sure this incorrect spelling is used else where. Could you open an issue for this?